products/compute/api.yaml

```diff
@@ -243,8 +245,9 @@ objects:
                 Provide this property when you create the resource.
             - !ruby/object:Api::Type::ResourceRef
               name: 'group'
-              resource: 'InstanceGroup'
-              imports: 'selfLink'
+              resources:
+                - resource: 'InstanceGroup'
```

Comment:
How come these aren't !ruby/object:Api::Type::IndividualResourceRef?

Reply:
Maybe we should discuss that? I debated back and forth on whether or not I should do that. I thought having just a dictionary would be simpler. The idea was that there wouldn't be a need (at the moment) for ResourceRef to take in anything but an IndividualResourceRef. That means that adding `!ruby/object` just ends up being boilerplate. Thoughts?

Reply:
I prefer it this way - it still throws validation errors when there are extra fields (or a required field is missing), right?